File: API/Entities/Promotion.cs
namespace API.Entities
{
    public class Promotion
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public bool Public { get; set; }
        public DateTime Created { get; set; } = DateTime.Now;
        public decimal Price { get; set; }
        public string? City { get; set; }
        public string? Month { get; set; }
        public ICollection<PromotionPhoto> PromotionPhotos { get; set; }
        public ICollection<PromotionDescription> PromotionDescriptions { get; set; }

    }
}

File: client/src/app/_models/promotion.ts
import { PromotionPhoto } from './promotionPhoto';

export interface Promotion{
  id:number;
  name:string;
  description:string;
  public:boolean;
  photoUrl:string;
  price:number;
  created:Date;
  city:string;
  bestMonths:string;
  promotionPhotos:PromotionPhoto[];

}
